src/DependencyInjection/Configuration.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+namespace SumoCoders\FrameworkCoreBundle\DependencyInjection;
+
+use Symfony\Component\Config\Definition\Builder\TreeBuilder;
+use Symfony\Component\Config\Definition\ConfigurationInterface;
+
+class Configuration implements ConfigurationInterface
+{
+    public function getConfigTreeBuilder()
+    {
+        $treeBuilder = new TreeBuilder('sumo_coders_framework_core');
+
+        $treeBuilder->getRootNode()
+            ->children()
+                ->variableNode('content_security_policy')
+                    ->defaultValue([
+                        'default-src' => [
+                            "'self'",   // Default rule: only allow content from our own domain
+                        ],
+                        'style-src' => [
+                            "'self'",
+                            'https://fonts.googleapis.com', // Allow Google Fonts
```

Comment:
De Google Font URLs zouden er al uit moeten. De PR van Katrien om lokale fonts te gebruiken is al gemerged.

Reply:
Fixed in baf26a52dbafb11b368bd91c6e41e67dc3c6c3cb